lib/logstash/outputs/elasticsearch/http_client_builder.rb

```diff
@@ -131,5 +149,10 @@ def self.setup_basic_auth(logger, params)
         :password => unsafe_escaped_password
       }
     end
+
+    private
+    def self.sanitize_url(url)
```

Comment:
Sure, I used sanitize as "making sane", or "removing craziness", but I'll rename as you suggested